app/src/main/java/com/yterletskyi/happyfriend/features/contacts/ui/ContactsFragment.kt

```diff
@@ -51,6 +51,23 @@ class ContactsFragment : BaseBindingFragment<FragmentContactsBinding>(
             }
         }
 
+        with(binding.etSearch) {
+            binding.etClear.visibility = View.INVISIBLE
+            doAfterTextChanged {
+                if (text?.isEmpty() == false) {
```

Comment:
please use `it` (the `doAfterTextChanged ` parameter), and not `text` (the `with(binding.etSearch)` parameter)